### ISSUE TITLE
predict results on real documents is bad

### ISSUE THREAD

**Issue by huyutao3550346** [jottue/HTSNet#2]
hello author, I have implemented the HTSNet by keras from the paper. 
after train with 90 epochs  and 30 steps IOU is nearby 95%
and model predict well in synthesized test set. like:
origin image:
![4_syn](https://user-images.githubusercontent.com/29395068/114138588-6e27f080-9940-11eb-86b6-635c212ce011.jpg)
result image:
![4_syn jpg_out](https://user-images.githubusercontent.com/29395068/114138656-8861ce80-9940-11eb-935b-a9386d06c8c6.png)

but predict results on real documents is bad. like:
origin image:
![4](https://user-images.githubusercontent.com/29395068/114140860-6584e980-9943-11eb-963e-b6f9e8714fca.png)
result image:
![4 png_out](https://user-images.githubusercontent.com/29395068/114140916-79305000-9943-11eb-86f5-b18def2f5852.png)

here is my model define and train file:
model define:
https://drive.google.com/file/d/1XvIQswjj4iDwEowWw51eS1lOEPtjee_n/view?usp=sharing
predict.py:
https://drive.google.com/file/d/1aeLbIippxarnMvjEIjGie-0cx6n0SvP1/view?usp=sharing
train file:
https://drive.google.com/file/d/1V9IhUozXp_dEIO8boV9VNwbUbMqou3ea/view?usp=sharing

Is my loss function and model defined wrong or do I need to use real documents for training?
Do you have any good suggestions to solve this problem? thank you.
